src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
@arun-121 we are not testing this app right?, we can remove this unused test file

Reply:
removed the file